buildsystem/simple

```diff
@@ -15,9 +15,10 @@
 PYVER=3.4m
 PYVERDOTLESS=34m
 shopt -s globstar
+set -e
```

Comment:
I think this one should stay: we don't want to keep building if a previous step failed.